docs/proposals/20240807-in-place-updates.md

```diff
@@ -263,13 +272,11 @@ sequenceDiagram
 
 Both `KCP` and `MachineDeployment` controllers follow a similar pattern around updates, they first detect if an update is required and then based on the configured strategy follow the appropiate update logic (note that today there is only one valid strategy, `RollingUpdate`).
 
-With `ExternalUpdate` strategy, CAPI controllers will compute the set of desired changes and iterate over the registered external updaters, requesting through the Runtime Hook the set of changes each updater can handle. The changes supported by an updater can be the complete set of desired changes, a subset of them or an empty set, signaling it cannot handle any of the desired changes.
-
-On each iteration, the controller will substract from the set of desired changes the subset returned by the updater. 
+With `InPlaceUpdates` feature gate enabled, CAPI controllers will compute the set of desired changes and iterate over the registered external updaters, requesting through the Runtime Hook the set of changes each updater can handle. The changes supported by an updater can be the complete set of desired changes, a subset of them or an empty set, signaling it cannot handle any of the desired changes.
 
-If this set is reduced to zero, then CAPI will determine that the update can be performed using the external strategy. CAPI will define the update plan as a list of sequential external updaters in a particular order and proceed to execute it. The update plan will be stored in the Machine object as an array of strings (the names of the selected external updaters).
+If any updater can handle the desired changes then CAPI will determine that the update can be performed using the external strategy. 
```

Comment:
nit: maybe this could be misleading? It's not really any updater but "any combination of the updaters". wdyt?

Reply:
yes, thanks, that sounds better